indigo_app/templates/indigo_api/_work_points_in_time.html

```diff
@@ -30,157 +30,53 @@
-              {% trans 'Commenced' %}
-            {% else %}
-              {% trans 'Commenced in part' %}
-            {% endif %}
-            {% if commencement.commencing_work %}
-              {% trans 'by' %}
-              <a href="{% url 'work' frbr_uri=commencement.commencing_work.frbr_uri %}" data-popup-url="{% url 'work_popup' frbr_uri=commencement.commencing_work.frbr_uri %}">{{ commencement.commencing_work.title }}</a>
-            {% endif %}
-          </h6>
-          {% if commencement.commencing_work %}
-            <div class="text-muted mb-3">{{ commencement.commencing_work.frbr_uri }}</div>
-          {% endif %}
-          {% if commencement.note %}
-            <p class="mt-3">
-              {% trans 'Note:' %} {{ commencement.note }}
-            </p>
-          {% endif %}
-        {% endfor %}
-
-        {% for consolidation in event.consolidations %}
-          {% if consolidation.pk %}
-            <form action="{% url 'edit_arbitrary_expression_date' frbr_uri=work.frbr_uri arbitrary_expression_date_id=consolidation.id %}" method="POST" id="arbitrary-expression-date-{{ consolidation.id }}">
-              {% csrf_token %}
-
-              {% if editable and perms.indigo_api.change_arbitraryexpressiondate %}
-                <div class="dropdown float-right">
-                  <button class="btn btn-outline-secondary btn-sm dropdown-toggle" type="button" data-toggle="dropdown">
-                    {% trans 'Options' %} <span class="caret"></span>
-                  </button>
-                  <div class="dropdown-menu">
-                    <a href="#arbitrary-expression-date-form-{{ consolidation.id }}" class="dropdown-item" data-toggle="collapse">{% trans 'Change date' %}</a>
-                    {% if consolidation.can_delete %}
-                      <button class="dropdown-item" type="submit" name="delete" data-confirm="{% trans 'Really delete this expression date?' %}">{% trans 'Delete' %}</button>
-                    {% endif %}
-                  </div>
-                </div>
-              {% endif %}
-
-              <h6>{% trans 'Consolidation' %}</h6>
-              <div class="form-inline edit-wrapper mb-3 collapse" id="arbitrary-expression-date-form-{{ consolidation.id }}">
-                <input type="text" class="form-control amendment-date mr-2" data-provide="datepicker" placeholder="yyyy-mm-dd" pattern="\d{4}-\d\d-\d\d" required name="date" value="{{ consolidation.date|date:"Y-m-d" }}" autocomplete="off">
-                <button class="btn btn-success" type="submit">{% trans 'Save' %}</button>
-                <a href="#arbitrary-expression-date-form-{{ consolidation.id }}" class="btn btn-link" data-toggle="collapse">{% trans 'Cancel' %}</a>
-              </div>
-            </form>
+        {% for event in entry.events %}
+          {% if editable and event.related and event.type == 'amendment' %}
+            {% include 'indigo_api/_edit_amendment.html' %}
```

Comment:
Perhaps move these included templates under `indigo_api/timeline/` so that it's clear they're scoped to the timeline. We haven't done that much in indigo, and it means the `indigo_api` folder has a huge number of templates.